user-guide/knative.md

```diff
@@ -1,101 +1,24 @@
 # Using Knative and Ambassador
 
-[Knative](https://knative.dev/) is a popular Kubernetes-based platform for managing serverless workloads with 2 main components:
-- Eventing - Management and delivery of events
-- Serving - Request-driven compute that can scale to zero
+[Knative](https://knative.dev/) is a popular Kubernetes-based platform for managing serverless workloads with two main components:
+- Eventing: Management and delivery of events
+- Serving: Request-driven compute that can scale to zero
 
-We will be focusing on Knative Serving which builds on Kubernetes to support deploying and serving of serverless applications and functions.
+We will be focusing on Knative Serving, which builds on Kubernetes to support deploying and serving of serverless applications and functions.
 
-The Ambassador Edge Stack can watch for changes in Knative configuration in your Kubernetes cluster and set up routing accordingly.
+Ambassador can watch for changes in Knative configuration in your Kubernetes cluster and set up routing accordingly.
 
-**Note:** Knative was originally built with Istio handling cluster networking. This integration lets us replace Istio with the Ambassador Edge Stack which will dramatically reduce the operational overhead of running Knative.
+**Note:** Knative was originally built with Istio handling cluster networking. This integration lets us replace Istio with  Ambassador, which will dramatically reduce the operational overhead of running Knative.
 
 ## Getting started
 
 #### Prerequisites
 
 - Knative now requires Kubernetes v1.14, as well as a [compatible kubectl](https://knative.dev/docs/install/knative-with-ambassador/)
 - `kubectl` v1.10 is also required. This guide assumes that you’ve already created a Kubernetes cluster which you’re comfortable installing alpha software on.
-- The Ambassador Edge Stack should be installed in your cluster. Follow the [installation guide](../install) for instructions on installing the Ambassador Edge Stack.
 
 #### Installation
 
-1. Install Knative:
+Install the latest Knative Serving with Ambassador handling traffic to your serverless applications by following the instructions [here](https://knative.dev/docs/install/knative-with-ambassador/).
```

Comment:
@concaf this is the only place I had a question on. Could we change it to say "Serving with Ambassador to handle traffic"? Not sure if that maintains the technical accuracy or not.

Reply:
@scoyle391 sure, we can change it to - `Install the latest Knative Serving with Ambassador to handle traffic to your serverless applications by following the instructions`

Sorry, but I couldn't find any comment from you earlier in the PR :( maybe due to new changes. Nvm.